style.css

```diff
@@ -0,0 +1,55 @@
+body {
+  font-family: Arial, sans-serif;
+}
+
+form {
+  max-width: 600px;
+  margin: 0 auto;
+  padding: 20px;
+  border: 1px solid #ccc;
+  border-radius: 10px;
+}
+
+h1 {
+  text-align: center;
+  margin-top: 0;
+}
+
+label {
+  display: block;
+  margin-bottom: 10px;
+}
+
+.input-field {
```

Comment:
select field and input field have the same css this works fine but as a best practice combine i guess

Reply:
- Following the article on how to give a review, it suggests Using emojis in your review, I'll also suggest you edit your review comment and add a friendly emoji, this will make the review more human-friendly. :+1: :hugs: :+1: